transformers/ArbitrumPollingTransformer.js

```diff
@@ -6,18 +6,13 @@ const {
 } = require('@makerdao-dux/spock-utils/dist/extractors/rawEventDataExtractor');
 const { getLogger } = require('@makerdao-dux/spock-etl/dist/utils/logger');
 const BigNumber = require('bignumber.js').BigNumber;
+const ethers = require('ethers');
```

Comment:
^can delete this import now. Otherwise looks good!

Reply:
oops, my linter was not turned on for this workspace 🤦